tests/test_vprec_backend/generic_test.sh

```diff
@@ -31,7 +37,7 @@ float_type_list=("float" "double")
 
 # Modes list
 if [ $USECASE = "fast" ]; then
-    modes_list=("OB")
+    modes_list=("FULL")
```

Comment:
As we discuss during last friday meeting, maybe we can set to OB by default for the fast test in order to save testing time on PR push. 
The rational is that OB alone is enough to validate the rounding functions.
However full is also testing the computation part of the backend, which is bugfree and not suppose to move, but who knows... 
In a nutshell I am ok to go back on "OB". Anyobjection from the other PR contributors?

Reply:
done